app/src/main/res/xml/privacy_preferences.xml

```diff
@@ -6,20 +6,20 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <PreferenceCategory
-        android:title="@string/tracker_category">
+    <!--<PreferenceCategory-->
```

Comment:
how about a block comment so it introduce less merge conflicts on syncing with upstream?